marklogic-client-api/src/main/java/com/marklogic/client/dataservices/CallBatcher.java

```diff
@@ -33,6 +33,12 @@
     CallBatcher<W,E> withJobId(String jobId);
     CallBatcher<W,E> withJobName(String jobName);
     CallBatcher<W,E> withThreadCount(int threadCount);
+    
+    /**
+     * Adds the CallArgs to the current instance of CallBatcher.
```

Comment:
The default argument for a parameter is used only when the parameter isn't otherwise specified.

Would it be useful to explain this to the API user?  If one were using this API, what would be helpful to understand?

Reply:
Thanks Erik.! 
Adding the explanation for default args.